compiler/rustc_middle/src/ty/util.rs

```diff
@@ -1538,17 +1538,26 @@ pub fn is_doc_notable_trait(tcx: TyCtxt<'_>, def_id: DefId) -> bool {
         .any(|items| items.iter().any(|item| item.has_name(sym::notable_trait)))
 }
 
-/// Determines whether an item is an intrinsic by Abi.
-pub fn is_intrinsic(tcx: TyCtxt<'_>, def_id: LocalDefId) -> bool {
-    matches!(tcx.fn_sig(def_id).skip_binder().abi(), Abi::RustIntrinsic | Abi::PlatformIntrinsic)
+/// Determines whether an item is an intrinsic by Abi. or by whether it has a `rustc_intrinsic` attribute
```

Comment:
```suggestion
/// Determines whether an item is an intrinsic (which may be via Abi or via the `rustc_intrinsic` attribute)
```

Reply:
(The reason for my suggestion is that the doc as written, interpreted literally, would mean that the method will tell you *which way* the item is an intrinsic. But of course this is not what the method does, it doesn't have enough info in its return type to indicate that. Still I figured its best to just try to make the english as clear as possible.)